src/Benchmarks/Middleware/PlaintextMiddleware.cs

```diff
@@ -26,12 +26,13 @@ public Task Invoke(HttpContext httpContext)
         {
             if (httpContext.Request.Path.StartsWithSegments(_path, StringComparison.Ordinal))
             {
-                httpContext.Response.StatusCode = 200;
-                httpContext.Response.ContentType = "text/plain";
+                var response = httpContext.Response;
+                response.StatusCode = 200;
+                response.ContentType = "text/plain";
                 // HACK: Setting the Content-Length header manually avoids the cost of serializing the int to a string.
                 //       This is instead of: httpContext.Response.ContentLength = _helloWorldPayload.Length;
-                httpContext.Response.Headers["Content-Length"] = "13";
-                return httpContext.Response.Body.WriteAsync(_helloWorldPayload, 0, _helloWorldPayload.Length);
+                response.Headers["Content-Length"] = "13";
+                return response.Body.WriteAsync(_helloWorldPayload, 0, _helloWorldPayload.Length);
```

Comment:
Isn't this going through the `HttpResponse` not the `IHttpResponseFeature`? I think there would be more we could shave off using the feature.